src/modules/Dropdown/Dropdown.js

```diff
@@ -849,9 +863,11 @@ export default class Dropdown extends Component {
         active={isActive(opt.value)}
         onClick={this.handleItemClick}
         selected={selectedIndex === i}
-        onMouseDown={e => e.preventDefault()} // prevent default to allow item select without closing on blur
+        // prevent default to allow item select without closing on blur
+        onMouseDown={e => e.preventDefault()}
```

Comment:
I think you can get rid of this now since `preventDefault` here is meant to prevent closing on blur but you're preventing that more completely via this PR.